nailgun/entity_mixins.py

```diff
@@ -1245,6 +1261,9 @@ def search(self, fields=None, query=None, filters=None):
         ]
         if filters is not None:
             entities = self.search_filter(entities, filters)
+
+        signals.post_search.send(self, entities=entities, fields=fields,
+                                 query=query, filters=filters)
```

Comment:
I imagine a scenario where one would like to apply a filter to every search in batch,

```python
@signals.pre_search.connect
def add_filters_to_every_search(sender, fields, query, filters):
    filters.append(#filters_here)
```

With that any **.search** performed will automatically get that filters added.

The post_search may be useful to record logs, statistics etc

There is no overhead on emitting signals, but it looks like uncommon scenarios, if agreed I can remove the search signals.

```